lsp-metals.el

```diff
@@ -635,6 +640,11 @@ FOCUSED if there is a focused frame."
                                   cl-third
                                   string-to-number))))))
 
+(defun lsp-metals-run-scalafix-rules-on-save ()
```

Comment:
Ok, I no longer think running on save is a good idea :sweat_smile: , with rules like remove unused it can be really annoying, will remove